website/content/en/docs/setup/going-to-prod/sizing.md

```diff
@@ -0,0 +1,154 @@
+
+| Cloud | Recommendations |
+| --- | --- |
+| AWS | EBS io2, 10m worth of data |
+| Azure | Ultra-disk or standard SSD, 10m worth of data |
+| GCP | Balanced or SSD persistent disks, 10m worth of data |
+
+{{< info >}}
+More information about buffers and data loss can be found in the [high availability document](/docs/setup/going-to-prod/high-availability/).
+{{< /info >}}
+
+## Scaling
+
+### Vertical Scaling
+
+[Vector’s concurrency model](/docs/about/under-the-hood/architecture/concurrency-model/) automatically scales to take advantage of all vCPUs. There are no configuration changes needed. When vertically scaling, we recommend capping an instance’s size to process no more than 33% of your total volume. This allows for [high availability](/docs/setup/going-to-prod/high-availability/) in the event of a node failure.
+
+{{< info >}}
+More information about vertical sizing can be found in the [capacity planning section](#capacity-planning).
+{{< /info >}}
+
+### Horizontal Scaling
+
+Fronting your Vector instances with a load balancer achieves horizontal scaling.
+
+![Load balancing example](/img/going-to-prod/horizontal-scaling.png)
+
+#### Choosing a Load Balancer
+
+Choose the load balancer you’re most comfortable operating in a highly available manner. When in doubt, use a managed load balancer, such as AWS ALB. These load balancers are highly available and easier to integrate within your cloud environment.
+
+#### Configuring a Load Balancer
+
+When configuring a load balancer, we recommend the following general settings:
+
+- [Choose a protocol](/docs/setup/going-to-prod/architecting/#protocols) between your clients that allows for even load-balancing and application-level acknowledgement. Such as an HTTP-based protocol.
+- Enable keep-alive for both clients and servers.
+- If you’re using stateful transforms, such as the `aggregate` or `dedupe` transforms, use a load balancing algorithm that consistently routes clients to the same server (i.e., the HAProxy `source` algorithm).
+- If you’re not using stateful transforms, use a load balancing algorithm that evenly distributes traffic (i.e., the HAProxy `roundrobin` algorithm).
+- Configure load balancers to use Vector’s `/health` API endpoint for automatically excluding unresponsive Vector servers.
+- Ensure that load balancers automatically register all targets as your aggregators scale up and down.
+
+#### Avoiding Hot Spots
+
+Not all connections are equal; some connections produce much more data making it difficult to evenly load balance traffic across your aggregators. To mitigate this, we recommend the following best practices:
+
+- Use a protocol that allows for even load balancing, such as an HTTP-based protocol. Avoid plain TCP or UDP connections for the same reasons.
```

Comment:
I think `udp` is fairly easily load balanced. I might update this to just say `TCP`.